chore: openreports blogpost (#328)
kubewarden/kubewarden.io: content/blog/2025/04/openreport.md

---
title: Kubewarden joins OpenReports
Initiative
authors:
  - José Guilherme Vanz

> date: 2025-04-17

Author: Update this date before merging

---

We in the Kubewarden project, as an Open Source and CNCF project, actively engage
with the broader Kubernetes ecosystem. This commitment drives our decision to reuse

> valuable projects like Policy Reporter. By adopting it as our default UI for Kubewarden

Reviewer: or our documentation at https://docs.kubewarden.io/explanations/audit-scanner/policy-reports#policy-reporter-ui

> We in the Kubewarden project, as an Open Source and CNCF project, actively engage
> with the broader Kubernetes ecosystem. This commitment drives our decision to reuse
> valuable projects like Policy Reporter. By adopting it as our default UI for Kubewarden
> audit scanner results, we simplify the user experience by reducing the learning curve
> and enable them to leverage familiar reporting mechanisms. This strategic choice
> allows our team to dedicate more resources to developing impactful features within
> our core stack.

Reviewer 2: ```suggestion
Kubewarden is an open-source CNCF project actively engaged with the wider Kubernetes ecosystem. This informs our use of valuable projects like Policy Reporter. Using Policy Reporter as a default UI for Kubewarden simplifies the user experience, allowing the use of familiar reporting mechanisms. This strategic choice also allows the team to concentrate on the Kubewarden core stack.
```

> With this in mind, the Kubewarden team decided to move one step forward and start
> to actively participate in the [Kubernetes Policy Working Group](https://github.com/kubernetes/community/blob/master/wg-policy/README.md)
> to shape the future of policy enforcement in the Kubernetes landscape. As a first step,
> we began by joining the community meetings and proactively seeking opportunities
> to work together. Early in these discussions, the community focused on the future
> of the policy report and related resources created within the working group. After
> some deliberation, the community decided to move the policy report to its own
> independent project, giving birth to the [OpenReports](https://openreports.io/) project!

Reviewer 2: ```suggestion
So, the Kubewarden team participates in the 
[Kubernetes Policy Working Group](https://github.com/kubernetes/community/blob/master/wg-policy/README.md). We join community meetings and seek opportunities for collaboration, focusing on the future of policy reporting and related resources. The community moved policy reporting to an independent project, the [OpenReports](https://openreports.io/) project!
```

> Inspired by OpenTelemetry's unifying role for metrics and traces, the OpenReports
> project aims to provide a unified API to create and consume diverse reports in the
> Kubernetes ecosystem, expanding the original goal of policy engines like Kubewarden.
> The Kubewarden team has been involved since the beginning of this initiative and
> will continue to collaborate with the community to build a more robust and integrated
> ecosystem.

Reviewer 2: ```suggestion
Inspired by OpenTelemetry's unifying approach to metrics and traces, the OpenReports project provides an API for creating and consuming reports in the Kubernetes ecosystem. This benefits Kubewarden and other projects. Kubewarden has been involved since the beginning of OpenReports, and we will continue to help the community build a robust, integrated ecosystem.
```

> You can learn more about OpenReports in its official project announcement!

Author: Wait for the official announce.

> You can learn more about OpenReports in its official project announcement!

Reviewer 2: ```suggestion
You can learn more about OpenReports in the official project announcement!
```
